README.md

```diff
@@ -275,8 +275,7 @@ Each group consists of:
 - a mapping of InSpec attribute names to Terraform output variable
   names to define for the suite's InSpec profile. By default, all Terraform
```

Comment:
Mentioning the configuration file seems redundant since this section is discussing the file's options.